notebook/tests/selenium/test_dashboard_nav.py

```diff
@@ -3,43 +3,70 @@
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support.ui import WebDriverWait
 from selenium.webdriver.support import expected_conditions as EC
-
 pjoin = os.path.join
 
+
+class PageError(Exception):
+    """Error for an action being incompatible with the current jupyter web page.
+    
+    """
+    def __init__(self, message):
+        self.message = message
+        
+        
+
 def get_list_items(browser):
+    """Gets list items from a directory listing page
+    
+    Raises PageError if not in directory listing page (url has tree in it)
+    """
+    try:
+        assert 'tree' in browser.current_url
+    except PageError:
```

Comment:
This is a convoluted way of writing `if 'tree' not in browser.current_url:` , even if it was catching the right error. Let's stick to the simple if statement rather than using assert to achieve the same thing.

Reply:
fair enough — changed.